services/core/businessconfig/src/main/modeling/swagger.yaml

```diff
@@ -326,6 +326,43 @@ paths:
           description: Authentication required
         '403':
           description: Forbidden - ADMIN role necessary
+  '/businessconfig/monitoring':       
+    get:
+      summary: Get the monitoring configuration 
+      description: >-
+         Get the monitoring configuration 
+      operationId: getMonitoring
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: OK
+          schema:
+            $ref: '#/definitions/Monitoring'
+        '401':
+          description: Authentication required
+    post:
```

Comment:
Looking at the status codes got me thinking, is 201 (Successful creation) really appropriate when we're updating the configuration, not creating it? 
So I googled it and found several links, including [this one](https://www.moesif.com/blog/technical/api-design/Which-HTTP-Status-Code-To-Use-For-Every-CRUD-App/) that seems to indicate that in this case we should be using "PUT" rather than "POST" anyway, because we're not really creating a new resource with every post (as opposed to users or cards for example).
But the POST is coherent with the other existing endpoints of the same type (processGroups for example)... What do you think?

Reply:
we add an issue for all endpoint concerns (outside of the scope of this Pr)

Reply:
Created https://opfab.atlassian.net/browse/OC-1685